python/dask_cudf/dask_cudf/tests/test_core.py

```diff
@@ -938,3 +940,15 @@ def test_categorical_dtype_round_trip():
     actual = ds.compute()
     expected = pds.compute()
     assert actual.dtype.ordered == expected.dtype.ordered
+
+
+def test_object_to_string_fail(request):
+    request.applymarker(
+        pytest.mark.xfail(
+            reason="https://github.com/rapidsai/cudf/issues/14915",
+        )
+    )
+    s = cudf.Series(["a", "b", "c"] * 10)
+    ds = dgd.from_cudf(s, npartitions=2)
+    pds = dd.from_pandas(s.to_pandas(), npartitions=2)
+    dd.assert_eq(ds.sort_values(), pds.sort_values())
```

Comment:
@galipremsagar - I started looking into #14915 and found that this test actually does **not** fail because of string conversion.

You cannot call `sort_values` on a `Series`. The code would need to use a `DataFrame` like the following (which does not fail anymore btw):

```python
    s = cudf.DataFrame({"x": ["a", "b", "c"] * 10})
    ds = dgd.from_cudf(s, npartitions=2)
    pds = dd.from_pandas(s.to_pandas(), npartitions=2)
    dd.assert_eq(ds.sort_values("x"), pds.sort_values("x"))
```